### ISSUE TITLE
CameraDescription does not accept different versions of CameraGeometry

### ISSUE THREAD

**Issue by rlopezcoto** [cta-observatory/cta-lstchain#452]
https://github.com/cta-observatory/ctapipe/pull/1241 refactored the code for `CameraGeometry`, including it inside the class `CameraDescription`. This was one of the points we wanted to update when moving to ctapipe v0.8 (#357), but it turns out we cannot select different versions of the geometry, as it would be needed e.g. https://github.com/cta-observatory/cta-lstchain/blob/cba047818cdec88122aebcee242d3966d3aff300/lstchain/visualization/plot_drs4.py#L82

@watsonjj suggested one could do:
```
geom = CameraGeometry.from_name("LSTCam", 2)
readout = CameraReadout.from_name("LSTCam", 2) #  although this is not defined from our side
desc = CameraDescription(name, geom, readout)
```

or we could simply not move to `CameraDescription` until this gets solved and release in a new ctapipe release. Any opinions?

**Comment by MaxNoe** [comment]
I don't really understand this issue and the propositions made here.

`CameraDescription` is new in ctapipe 0.8 to also provide info of the electronics (e.g. reference pulse shape and sampling frequency), not only the pixel geometry. This information is used in several places (e.g. in calibration code).

So there is no way around this.

Furthermore, providing the correct subarray and thus telescope description is the job of the `EventSource`, this is in `ctapipe_io_lst`.

So what exactly is the issue here?

**Comment by MaxNoe** [comment]
In the example you linked, the correct thing to do is just to access `event_source.subarray` and not creating some CameraGeometry

**Comment by rlopezcoto** [comment]
as suggested, the correct way was to use `event_source.subarray` and it was implemented in #479
